commands/ps/type.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let cli  = require('heroku-cli-util');
+let co   = require('co');
+
+const costs = {"Free": 0, "Hobby": 7, "Standard-1X": 25, "Standard-2X": 50, "Performance-M": 250, "Performance": 500, "Performance-L": 500, "1X": 36, "2X": 72, "PX": 576};
+
+function* run (context, heroku) {
+  let _ = require('lodash');
+
+  let app = context.app;
+
+  let parse = co.wrap(function* (args) {
+    if (args.length === 0) return [];
+    let formation = yield heroku.get(`/apps/${app}/formation`);
+    if (args.find(a => a.match(/=/))) {
+      return _.compact(args.map(arg => {
+        let match = arg.match(/^([a-zA-Z0-9_]+)=([\w-]+)$/);
+        let type = match[1];
+        let size = match[2];
+        if (!formation.find(p => p.type === type)) {
+          throw new Error(`Type ${cli.color.red(type)} not found in process formation.\nTypes: ${cli.color.yellow(formation.map(f => f.type).join(', '))}`);
+        }
+        return {type, size};
+      }));
+    } else {
+      return formation.map(p => ({type: p.type, size: args[0]}));
+    }
+  });
+
+  let displayFormation = co.wrap(function* () {
+    let formation = yield heroku.get(`/apps/${app}/formation`);
+    formation = _.sortBy(formation, 'type');
+    formation = formation.map(d => ({
+      type:      cli.color.green(d.type),
+      size:      cli.color.cyan(d.size),
+      qty:       cli.color.yellow(d.quantity.toString()),
+      'cost/mo': costs[d.size] ? (costs[d.size] * d.quantity).toString() : '0',
```

Comment:
@dickeyxxx This should be blank, not zero.  If somehow we have a new thing added to the hash we would not want it to show up as zero cost, it would be better if it were blank, and match the old behavior.